airqo-platform/AirQo-frontend
# Update insights_tab.dart

#### Summary of Changes (What does this PR do?)
- Added a pop up menu in the insights tab on the toggle button that displays options of PM2.5 and PM10 to choose from.

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Fetch and checkout to this branch locally
* Run app to check for changes.
* Try using the toggle option in the insights tab.

#### What are the relevant tickets?
- [MOB-40](https://airqoteam.atlassian.net/browse/MOB-40)

#### Screenshots (optional)

![ScreenShot](https://user-images.githubusercontent.com/86492979/174249384-15849dd9-90cb-449c-8ad4-f02e5b7e236d.jpg)
